lib/cc/service.rb

```diff
@@ -69,10 +69,11 @@ def self.slug
       end
     end
 
-    def initialize(config, payload)
-      @payload = payload.stringify_keys
-      @config  = create_config(config)
-      @event   = @payload["name"].to_s
+    def initialize(config, payload, beta_service_features)
```

Comment:
Hmm. Not sure I like the name so much. How about something more along the lines of `feature_toggle` ? Whether its beta or not is sort of a separate concern.